src/script/script.cpp

```diff
@@ -238,11 +240,11 @@ bool CScript::IsPushOnly() const
     return true;
 }
 
-std::string CScript::ToString() const
```

Comment:
We're doing the opposite in other places. That doesn't mean we should do it here. If you were touching the lines anyway I wouldn't have said anything, but increasing the diff only for consistency doesn't seem very interesting in my opinion. Anyway, feel free to ignore, just a small nit.